.github/workflows/scaffold_pkg_via_pull_request_assignment.yml

```diff
@@ -135,12 +133,51 @@ jobs:
           # Create `scaffold/<path>` branch:
           git branch $SCAFFOLD_BRANCH origin/${{ github.event.pull_request.head.ref }}
 
+          # Checkout `scaffold/<path>` branch:
```

Comment:
@Planeshifter Did you mean to include these changes in this PR?

Reply:
Hmm, so I synced the forked branch to the upstream develop branch and now this is outdated. Maybe what happened is that the workflow was changed between the time when  the feature branch was created on the fork to when the scaffold branch on this repo was created? When merging in commits from upstream this will then include those changes, I think.